### PR TITLE
implement the method findProductListBrandRelationByIdProductList

### PR DESCRIPTION
**Pull request by gengjozsef**

### CODE REVIEW

File: src/FondOfSpryker/Zed/ProductListBrandConnector/Business/Model/ProductListBrandRelationReader.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+use FondOfSpryker\Zed\ProductListBrandConnector\Dependency\Facade\ProductListBrandConnectorToBrandProductFacadeInterface;
+use FondOfSpryker\Zed\ProductListBrandConnector\Dependency\Facade\ProductListBrandConnectorToProductListFacadeInterface;
+use Generated\Shared\Transfer\BrandRelationTransfer;
+use Generated\Shared\Transfer\ProductListTransfer;
+
+class ProductListBrandRelationReader implements ProductListBrandRelationReaderInterface
+{
+    /**
+     * @var \FondOfSpryker\Zed\BrandProduct\Business\BrandProductFacadeInterface
```
Comment (daniel-rose) [inline src/FondOfSpryker/Zed/ProductListBrandConnector/Business/Model/ProductListBrandRelationReader.php]: \FondOfSpryker\Zed\ProductListBrandConnector\Dependency\Facade\ProductListBrandConnectorToBrandProductFacadeInterface

File: src/FondOfSpryker/Zed/ProductListBrandConnector/Business/Model/ProductListBrandRelationReader.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        ProductListBrandConnectorToProductListFacadeInterface $productListFacade
+    ) {
+        $this->brandProductFacade = $brandProductFacade;
+        $this->productListFacade = $productListFacade;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\ProductListTransfer $productListTransfer
+     *
+     * @return \Generated\Shared\Transfer\BrandRelationTransfer
```
Comment (daniel-rose) [inline src/FondOfSpryker/Zed/ProductListBrandConnector/Business/Model/ProductListBrandRelationReader.php]: \Generated\Shared\Transfer\ProductListTransfer

File: src/FondOfSpryker/Zed/ProductListBrandConnector/Business/ProductListBrandConnectorFacade.php [hunk 3]
```diff
@@ -25,4 +25,20 @@ public function addBrandRelationToProductList(ProductListTransfer $productListTr
             ->createProductListExpander()
             ->addBrandRelationToProductListTransfer($productListTransfer);
     }
+
+    /**
+     * @inheritDoc
```
Comment (daniel-rose) [inline src/FondOfSpryker/Zed/ProductListBrandConnector/Business/ProductListBrandConnectorFacade.php]: {@inheritDoc}

**Review by daniel-rose — CHANGES_REQUESTED** [review]

File: src/FondOfSpryker/Zed/ProductListBrandConnector/Business/Model/ProductListBrandRelationReader.php (hunk 2 above)
Comment (gengjozsef) [reply src/FondOfSpryker/Zed/ProductListBrandConnector/Business/Model/ProductListBrandRelationReader.php]: fixed

File: src/FondOfSpryker/Zed/ProductListBrandConnector/Business/Model/ProductListBrandRelationReader.php (hunk 1 above)
Comment (gengjozsef) [reply src/FondOfSpryker/Zed/ProductListBrandConnector/Business/Model/ProductListBrandRelationReader.php]: fixed

File: src/FondOfSpryker/Zed/ProductListBrandConnector/Business/ProductListBrandConnectorFacade.php (hunk 3 above)
Comment (gengjozsef) [reply src/FondOfSpryker/Zed/ProductListBrandConnector/Business/ProductListBrandConnectorFacade.php]: fixed

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**